README.md

```diff
@@ -34,6 +34,18 @@ The following timings are supported and can be logged by `lambda-watchtower`:
 - `close`: Time between first byte and the end of the request
 - `total`: Total time from the beginning to the end of the request
 
+
+# Setup Guide
```

Comment:
Good idea to add the setup guide! I’ve actually already written one a while ago, I should have linked it here already: https://medium.com/@pentacent/serverless-monitoring-of-apps-websites-with-aws-lambda-5431e6713a66
You’ve briefly summed it up, so I think we can just add it like that. Maybe later it would be nice to expand it a bit and add the screenshots I’ve already made for the Medium article so it’s easier for newcomers to understand what they’re doing.

Reply:
Adding the detailed how to guide would be useful as well. I've seen other lambda function repos that add a cloudformation yaml that does all the config for the user as well (Lambstatus), which makes it one-click-drop-dead-simple.

Reply:
Oh, I like that idea. I’ve added issue #6 for this :+1: